azurerm/internal/services/frontdoor/frontdoor_resource.go

```diff
@@ -433,32 +433,78 @@ func resourceFrontDoor() *schema.Resource {
 							Optional: true,
 							Default:  0,
 						},
-						"custom_https_provisioning_enabled": {
-							Type:       schema.TypeBool,
-							Optional:   true,
-							Computed:   true,
-							Deprecated: "Deprecated in favour of `azurerm_frontdoor_custom_https_configuration` resource",
-						},
+						// if either of these are set in the main FrontDoor config we need to error out
+						// unfortunately we will need to force end users to use the azurerm_frontdoor_custom_https_configuration
+						// "custom_https_provisioning_enabled": {
+						// 	Type:       schema.TypeBool,
+						// 	Optional:   true,
+						// 	Computed:   true,
+						// 	Deprecated: "Deprecated in favour of `azurerm_frontdoor_custom_https_configuration` resource",
+						// },
 						"web_application_firewall_policy_link_id": {
 							Type:     schema.TypeString,
 							Optional: true,
 							// TODO: validation that this is a resource id
 						},
-						"custom_https_configuration": {
-							Type:       schema.TypeList,
-							Optional:   true,
-							Computed:   true,
-							MaxItems:   1,
-							Deprecated: "Deprecated in favour of `azurerm_frontdoor_custom_https_configuration` resource",
-							Elem: &schema.Resource{
-								Schema: schemaCustomHttpsConfiguration(),
+						// "custom_https_configuration": {
+						// 	Type:       schema.TypeList,
+						// 	Optional:   true,
+						// 	Computed:   true,
+						// 	MaxItems:   1,
+						// 	Deprecated: "Deprecated in favour of `azurerm_frontdoor_custom_https_configuration` resource",
+						// 	Elem: &schema.Resource{
+						// 		Schema: schemaCustomHttpsConfiguration(),
+						// 	},
+						// },
+					},
+				},
+			},
+
+			// Computed values
+			"explicit_resource_order": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"backend_pool_ids": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+						"frontend_endpoint_ids": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+						"routing_rule_ids": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+						"backend_pool_load_balancing_ids": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+						"backend_pool_health_probe_ids": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
```

Comment:
Isn't this effectively a dupe of what follows below?

Reply:
I'm not exactly sure what you are saying here, that might be true but I need more context...